src/neuroconv/tools/nwb_helpers/_configuration_models/_base_dataset_io.py

```diff
@@ -260,9 +260,10 @@ def from_neurodata_object(cls, neurodata_object: Container, dataset_name: Litera
             )
             compression_method = "gzip"
         elif dtype == np.dtype("object"):  # Unclear what default chunking/compression should be for compound objects
-            chunk_shape = None
-            buffer_shape = None
-            compression_method = None
+            raise NotImplementedError(
```

Comment:
The other tools don't expect `None` here, nor does it make sense if it's a field that's going to be grabbed this way

To avoid configuring such a dataset, it simply shouldn't be parsed in this model to begin with

Reply:
What is an example of something that could have this dtype? What about text?

Reply:
I see. Thanks for explaining.